acceptance-tests/src/test/resources/features/site/facets.feature

```diff
@@ -7,6 +7,7 @@ Feature: Faceted search
     Scenario: All expected facets are displayed
         Given I navigate to the "home" page
         When I search for "zzFacets_Testzz"
+        And I click on the accept cookie banner
```

Comment:
It might be worth adding the step to one of the generic calls like `I navigate to` or `I search for` if it is required for all tests. It's not really relevant for the scenario.

Reply:
Hi Jacob. Good suggestion. We will add this as a separate fix. Thanks a lot!

Reply:
👍